VERSION

```diff
@@ -0,0 +1 @@
+0.1.0
```

Comment:
So how does this VERSION file work? What uses it etc?

Reply:
currrently jenkins job sources this to add tags to docker images.
will check if we can use git tags, removing this for now.